core/src/mindustry/content/UnitTypes.java

```diff
@@ -701,7 +701,6 @@ public void load(){
             legSpeed = 0.1f;
             legLengthScl = 0.96f;
             rippleScale = 2f;
-            legSpeed = 0.2f;
```

Comment:
0.2 was the value used before, i suggest you remove the 0.1 one

Reply:
yes, remove 0.1, not 0.2